bin/test-example

```diff
@@ -50,7 +58,9 @@ if [ "$exampletype" = "success" ]; then
     # Implicit exit value: this is last thing in this path,
     # so the script's exit value = stack's exit value.
     # If statements are reordered, please preserve this property.
-    runstack "test"
+    if ! runstack "test"; then
+        exit 1
+    fi
```

Comment:
> three lines above

the three lines being mentioned are the comments - the comments must be removed.